src/data/faq_de.json

```diff
@@ -493,7 +493,7 @@
                         "anchor": "where_to_get",
                         "active": true,
                         "textblock": [
-                            "Seit die App zur Verfügung steht, können Sie sie im App Store von Apple bzw. im Google Play Store herunterladen."
+                            "Seit die App zur Verfügung steht, können Sie sie im <a href='https://apps.apple.com/de/app/corona-warn-app/id1512595757' target='_blank' rel='noopener noreferrer'>Apple App Store</a> bzw. im <a href='https://play.google.com/store/apps/details?id=de.rki.coronawarnapp' target='_blank' rel='noopener noreferrer'>Google Play Store</a> herunterladen."

```

Comment:
And for the German language entries:

- https://apps.apple.com/de/app/corona-warn-app/id1512595757?l=de

- https://play.google.com/store/apps/details?id=de.rki.coronawarnapp&hl=de

Reply:
fixed wording (+ see above for explanation about links)